integration-tests/maven/src/test/java/io/quarkus/maven/it/DevMojoIT.java

```diff
@@ -814,4 +814,10 @@ public void testResourcesFromClasspath() throws MavenInvocationException, IOExce
         // test that we don't get multiple instances of a resource when loading from the ClassLoader
         assertThat(DevModeTestUtils.getHttpResponse("/cp/resourcesCount")).isEqualTo("1");
     }
+
+    @Test
+    public void testThatDependencyInParentIsEvaluated() throws IOException, MavenInvocationException {
+        testDir = initProject("projects/multimodule-parent-dep");
+        runAndCheck();
```

Comment:
Should we also be asserting the response of `/hello-0`?

Reply:
We could. I don't see anything special about it though. @famod wdyt?

Reply:
It _was_ special before this fix. After this fix both paths should behave the same.

I don't think `/hello-0` would ever break _individually_ in the future (`/hello` would then break as well, I suppose).

Long story short: I 'd leave it this way.

Reply:
OK, thanks